### PR TITLE
updates pause image to 3.6

### PR DESCRIPTION
**Pull request by wgahnagl**
#### What type of PR is this?
/kind dependency-change

#### What this PR does / why we need it:
Updates pause image to 3.6 from 3.5 per https://github.com/kubernetes/kubernetes/pull/104711 

#### Which issue(s) this PR fixes:

part of https://github.com/kubernetes/kubernetes/issues/104890 

#### Does this PR introduce a user-facing change?

```release-note
bumps pause image to 3.6
```

### CODE REVIEW

File: docs/crio.8.md [hunk 1]
```diff
@@ -1,422 +1,55 @@
...
-**storage.conf** (/etc/containers/storage.conf)
-  Storage configuration file specifies all of the available container storage
-  options for tools using shared container storage.
-
-# SEE ALSO
-
-crio.conf(5), crio.conf.d(5), oci-hooks(5), policy.json(5), registries.conf(5),
-storage.conf(5)
\ No newline at end of file
+Incorrect Usage. flag provided but not defined: -d
```
Comment (mrunalp) [inline docs/crio.8.md:1]: The changes in this file don't look related to the change.

**Comment by haircommander** [comment]
can you add a release note?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by mrunalp** [comment]
Could you add/update the release note in the first comment?
